is/jsonable_bench.ts

```diff
@@ -0,0 +1,94 @@
+import { assert } from "@std/assert";
+import { isJsonable } from "./jsonable.ts";
+
+const repeats = Array.from({ length: 100 });
+const testcases: [name: string, value: unknown][] = [
+  undefined,
+  null,
+  "",
+  0,
+  true,
+  [],
+  {},
+  0n,
+  () => {},
+  Symbol(),
+].map((x) => {
+  const t = typeof x;
+  switch (t) {
+    case "object":
+      if (x === null) {
+        return ["null", x];
+      } else if (Array.isArray(x)) {
+        return ["array", x];
+      }
+      return ["object", x];
+  }
+  return [t, x];
+});
+
+for (const [name, value] of testcases) {
+  switch (name) {
+    case "undefined":
+    case "bigint":
+    case "function":
+    case "symbol":
+      Deno.bench({
+        name: "current",
+        fn() {
+          assert(repeats.every(() => !isJsonable(value)));
+        },
+        group: `isJsonable (${name})`,
+      });
+      break;
+    default:
+      Deno.bench({
+        name: "current",
+        fn() {
+          assert(repeats.every(() => isJsonable(value)));
+        },
+        group: `isJsonable (${name})`,
+      });
+  }
+}
+
+for (const [name, value] of testcases) {
+  switch (name) {
+    case "undefined":
+    case "null":
+      continue;
+    case "bigint":
+    case "function":
+      Deno.bench({
+        name: "current",
+        fn() {
+          const v = Object.assign(value as NonNullable<unknown>, {
+            toJSON: () => "custom",
+          });
+          assert(repeats.every(() => isJsonable(v)));
+        },
+        group: `isJsonable (${name} with own toJSON method)`,
+      });
+  }
```

Comment:
`Object.assign()` changes test case value (object values).
Change to each test block generate the test case values.

The proto bench below is not working properly.

Reply:
Oops.